tutorials/Channel.dispatch.md

```diff
@@ -0,0 +1,25 @@
+Messages can be dispatched from the channelProvider to any channelClient or from any channelClient to the ChannelProvider. This messaging mechanism works exactly the same when messages are dispatched from the provider to a client.
+
+### Channel Provider
+Dispatch an action to a specified client. Returns a promise for the result of executing that action on the client side.
+
+```js
+(async ()=> {
+    const provider = await fin.InterApplicationBus.Channel.create('channelName');
+
+    await provider.onConnection(async (identity, payload) => {
+        await provider.dispatch(identity, 'welcome-action', 'Hello, World!');
```

Comment:
Wouldn't this prevent the client from ever connecting? dispatch would fail thus rejecting the connection

Reply:
my demo app printed the results, will change to that.